### PR TITLE
feat(Image): support base64 and multiple image sources

### PR DESCRIPTION
**Pull request by phongcao**

### CODE REVIEW

**Comment by rozele** [comment]
@phongcao if possible, keep future reviews separated by feature (in case one feature is good and the other needs work).

File: ReactWindows/ReactNative/UIManager/PropertySetter.cs [hunk 1]
```diff
@@ -114,8 +115,15 @@ protected object ExtractProperty(ReactStylesDiffMap props)
                 return defaultFunc(_attribute);
             }
 
-            return props.GetProperty(Name)?
-                .ToObject(PropertyType);
+            var prop = props.GetProperty(Name);
+            if (prop != null && prop.Type == JTokenType.Array)
```
Comment (rozele) [inline ReactWindows/ReactNative/UIManager/PropertySetter.cs]: This is going to break all array-based deserialization.  I imagine there's a better spot to fix this.

File: ReactWindows/ReactNative/Views/Image/ReactImageManager.cs [hunk 2]
```diff
@@ -205,6 +224,20 @@ public override void OnDropViewInstance(ThemedReactContext reactContext, Border
...
+        /// </summary>
+        /// <param name="view">The view.</param>
+        /// <param name="dimensions">The output buffer.</param>
+        public override void SetDimensions(Border view, Dimensions dimensions)
+        {
+            Canvas.SetLeft(view, dimensions.X);
+            Canvas.SetTop(view, dimensions.Y);
+            view.Width = dimensions.Width;
+            view.Height = dimensions.Height;
+            SetUriFromMultipleSources(view);
```
Comment (rozele) [inline ReactWindows/ReactNative/Views/Image/ReactImageManager.cs]: There's another lifecycle event we could use for this (OnAfterUpdate or something)

**Comment by phongcao** [comment]
Closed. Creating two separated PRs for this.

**Pull request closed without merging**